### ISSUE TITLE
minor issue in read.me file

### ISSUE THREAD

**Issue by axelbrunger** [hjdhjd/homebridge-doorbird#70]
Just installed the latest version of the doorbird plugin. It worked great when using the settings in the GUI. However, the read.me file is not complete. The first "name" statement is missing in the read.me file, and homebridge would not start. I then used the GUI to regenerate the statements in the config.json file, and it generates:
      {
            "name": "Doorbird",
            "audio": true,
            "doorbirds": [
                {
                    "name": "DoorBird",
                    "ip": "xxx",
                    "username": "xxx",
                    "password": "xxx"
                }
            ],
            "platform": "Doorbird"
        }

In other words, there have to be "name" statements in both the "platform" and the "doorbirds" levels.  
This configuration works!

**Comment by hjdhjd** [comment]
Thanks for reporting this, but I can't replicate your error.

You don't need the name field in any portion of the plugin. Perhaps you had a typo in your config.json and that's why it wouldn't start?

**Comment by axelbrunger** [comment]
Perhaps. Not sure. But the GUI generates both name fields…


On Jun 7, 2020, at 2:37 PM, hjdhjd <notifications@github.com<mailto:notifications@github.com>> wrote:



Thanks for reporting this, but I can't replicate your error.

You don't need the name field in any portion of the plugin. Perhaps you had a typo in your config.json and that's why it wouldn't start?

**Comment by hjdhjd** [comment]
Thanks. Closing this one out.
